io/disk/disk_info.py

```diff
@@ -130,9 +129,11 @@ def test(self):
         # Get the size and UUID of the disk
         cmd = "lsblk -l %s --output SIZE -b |sed -n 2p" % self.disk
         output = process.system_output(cmd, ignore_status=True,
-                                       shell=True, sudo=True)
-        self.size_bytes = (output.strip("\n"))[0]
-        self.log.info("Disk: %s Size: %s", self.disk, self.size_bytes)
+                                       shell=True, sudo=True).decode("utf-8")
+        if not output:
+            self.cancel("No information available in lsblk")
+        self.size_b = (output.strip("\n"))
```

Comment:
As output is already decoded as string, why is `[0]` element removed here. Is it a conscious change?

Reply:
Thanks for the catch. Was debugging, and missed it.